<launch>
  <arg name="robot_pose_topic" default="/spot_body_pose" />
  <arg name="region_size_x" value="10.0"/>
  <arg name="region_size_y" value="10.0"/>
  <arg name="region_size_z" value="2.4"/>

  <!-- publish local cloud -->
  <node name="spot_local_cloud_publisher"
        pkg="sloop_object_search_ros" type="local_cloud_publisher" output="screen">
    <param name="global_cloud_topic" value="/graphnav_map_publisher/graphnav_points"/>
    <param name="region_cloud_topic" value="region_points"/>
    <param name="robot_pose_topic" value="$(arg robot_pose_topic)"/>
    <param name="region_size_x" value="$(arg region_size_x)"/>
    <param name="region_size_y" value="$(arg region_size_y)"/>
    <param name="region_size_z" value="$(arg region_size_z)"/>
    <param name="retain_ratio" value="0.1"/>
  </node>
</launch>
